test/attributes.test.ts

```diff
@@ -49,16 +56,106 @@ describe("Attributes", () => {
         const getAttributesResponse = await client1.attributes.getValidAttributes({});
         expect(getAttributesResponse).toBeSuccessful(ValidationSchema.ConnectorAttributes);
     });
+
+    test("should create a Repository Attribute", async () => {
+        const attribute = await client1.attributes.createRepositoryAttribute({
+            content: {
+                value: {
+                    "@type": "GivenName",
+                    value: "AGivenName"
+                }
+            }
+        });
+        expect(attribute.isSuccess).toBe(true);
+    });
+
+    test("Should notify peer about Identity Attribute Succession", async () => {
+        const ownSharedRepositoryAttribute = await executeFullCreateAndShareIdentityAttributeFlow(client1, client2, {
+            "@type": "IdentityAttribute",
+            owner: client1Address,
+            value: {
+                "@type": "GivenName",
+                value: "AGivenName"
+            }
+        });
+
+        expect(ownSharedRepositoryAttribute.shareInfo?.sourceAttribute).toBeDefined();
+
+        const ownUnsharedRepositoryAttribute = await client1.attributes.getAttribute(ownSharedRepositoryAttribute.shareInfo!.sourceAttribute!);
+
+        const successionResponse = await client1.attributes.succeedAttribute(ownUnsharedRepositoryAttribute.result.id, {
+            successorContent: {
+                value: {
+                    "@type": "GivenName",
+                    value: "ANewGivenName"
+                }
+            }
+        });
+        expect(successionResponse.isSuccess).toBe(true);
+        const notificationResult = await client1.attributes.notifyPeerAboutIdentityAttributeSuccession(successionResponse.result.successor.id, {
```

Comment:
consistent variable naming throughout the tests: e. g. succeedAttribute returns a successionResponse, a succeedAttributeResponse and a result, notifyPeerAbout... returns a notificationResult